app/forms/register_user_email_form.rb

```diff
@@ -175,4 +177,8 @@ def existing_user
   def email_request_id(request_id)
     request_id if request_id.present? && ServiceProviderRequestProxy.find_by(uuid: request_id)
   end
+
+  def blocked_email_address
+    @blocked_email_address ||= SuspendedEmail.blocked_email_address(email)
```

Comment:
this can come back nil, so to memoize and prevent double queries we should do:
```suggestion
    return @blocked_email_address if defined?(@blocked_email_address)
    @blocked_email_address = SuspendedEmail.blocked_email_address(email)
```